src/app/students/(table)/columns.tsx

```diff
@@ -0,0 +1,46 @@
+"use client"
+
+import { Student } from "@/_temp_types/student"
+import { ColumnDef, Row } from "@tanstack/react-table"
+import { Badge } from "@/components/ui/badge"
+import { DataTableColumnHeader } from "@/components/ui/table-column-header"
+
+type SectionFilterValue = string[];
+
+export const columns: ColumnDef<Student>[] = [
+    {
+        id: "firstName",
+        header: ({ column }) => <DataTableColumnHeader column={column} title="First Name" hasDropDownMenu={false}/>,
+        accessorFn: (row) => row.name.split(',')[1],
+        cell: ({ getValue }) => <div className="text-left font-sm">{String(getValue())}</div>,
```

Comment:
complete nitpick, do as a separate PR, but these divs should probably be `<Text />` components